asteroid/losses/pit_wrapper.py

```diff
@@ -234,6 +235,10 @@ def find_best_perm(pair_wise_losses, n_src, perm_reduce=None, **kwargs):
         # Indexes and values of min losses for each batch element
         min_loss_idx = torch.argmin(loss_set, dim=1)
         min_loss, _ = torch.min(loss_set, dim=1, keepdim=True)
+
+        # Have a look here, would this be helful
+        batch_indices = torch.stack([perms[m] for m in min_loss_idx], dim=0)
```

Comment:
These will be of shape `(batch, n_src)` and give, for each batch, the optimal permutation on the sources.

Reply:
And actually, in `reorder_sources`, we re-create them (which is a bit stupid ^^)

Reply:
we return for each batch the optimal permutation `(batch, n_src) ` and `reorder_sources` will take the optimal permutation as argument. But this will require some re-factoring in a bunch of places

Reply:
```python
pw_losses = PITLossWrapper.get_pw_losses(lambda x, y: torch.sum((x.unsqueeze(1) * y.unsqueeze(2))),
                                                 current[..., :self.window_size//2],
                                                 previous[..., -self.window_size//2:])
_, perms = PITLossWrapper.find_best_perm(pw_losses, self.sources)
 current = PITLossWrapper.reorder_source(current, self.sources, perms)
```
It can be already done in this way but ideally we would want to have it in just two steps

Reply:
Yes, some place will need to be refactored but the output would be more useful in this way right?